docs/moderne-cli/README.md

```diff
@@ -117,35 +198,37 @@ Now that the repository is configured, it's time to migrate it to Spring Boot 3
        ▛▀▀█▀▛▀▀▀▀▜
        ▌▟▀  ▛▀▀▀▀▜
        ▀▀▀▀▀▀▀▀▀▀▀
-   Moderne CLI v0.3.2
+   Moderne CLI 1.0.0
 
    > Selecting repositories
 
-   > spring-projects/spring-petclinic@2.0.0
-   Selected 1 repositories (0.02s)
-   
+   > spring-projects/spring-petclinic@main
+   Selected 1 repositories
+   (0.54s)
    > Building LST(s)
-   
-   > spring-projects/spring-petclinic@2.0.0
-   Build output will be written to file:///Users/joanvr/workshop/spring-petclinic/.moderne/build/20230904223334-5kM1v/build.log
-   
-   🏗 Step 1 - build with Maven
-   Selected a Java 8 JDK.
-   
-   🏗 Step 2 - build with Native
-   ✅ Built LST file:///Users/joanvr/workshop/spring-petclinic/.moderne/build/20230904223334-5kM1v/spring-petclinic-20230904223403-ast.jar (28s)
-   
-   
-   MOD SUCCEEDED in (28s)             .
+
+   > spring-projects/spring-petclinic@main
+      Build output will be written to file:///Users/mikesol/Desktop/code/spring-petclinic/.moderne/build/20230921094850-s8Qfa/build.log
+      𓊍 Step 1 - build with Maven
+          Selected a Java 8 JDK.
+      𓊍 Step 2 - build resources using the native CLI
+      📈 Reported build metrics to Moderne
+      ✅ Built LST file:///Users/mikesol/Desktop/code/spring-petclinic/.moderne/build/20230921094850-s8Qfa/spring-petclinic-20230921094920-ast.jar
+   Built 1 repositories (30s)
+
+   * What to do next
+      > Run mod run . --recipe <RecipeName>
+
+   MOD SUCCEEDED in (30s)
    ```
    </details>
 
 
-2. Kick off the recipe by running the following command from the
+2. Kick off the migration recipe by running the following command from the
 `spring-petclinic` repository:
 
    ```shell
-   mod run . --recipe org.openrewrite.java.spring.boot3.VERSION_RECIPE_SPRING_BOOT
+   mod run . --recipe org.openrewrite.java.spring.boot3.UpgradeSpringBoot_3_1
```

Comment:
I'm not sure why we had `VERSION_RECIPE_SPRING_BOOT` before as that's not a recipe. I'd expect these commands to be ones where you can copy directly. Perhaps I'm missing something, though.

Reply:
It's part of running this script: https://github.com/moderneinc/springboot-migration-workshop/blob/main/src/scripts/versions.sh#L24
As briefly documented in the docs: https://github.com/moderneinc/springboot-migration-workshop/tree/main#develop

Reply:
Ah interesting - didn't realize there was some special build process that did this. Will update accordingly.